test/views/ReturnDeadlinesViewSpec.scala

```diff
@@ -47,7 +47,7 @@ class ReturnDeadlinesViewSpec extends ViewBaseSpec {
     val vatDetailsBreadCrumb = "div.govuk-breadcrumbs li:nth-of-type(2)"
     val vatDetailsBreadcrumbLink = "div.govuk-breadcrumbs li:nth-of-type(2) a"
 
-    val overdueLabel = ".task-overdue"
+    val overdueLabel = ".govuk-tag--red"
```

Comment:
Same as above in this spec. Could we add an additional test to the scenarios where a return is not overdue that checks the tag isn't present.